boot/zephyr/Kconfig

```diff
@@ -60,6 +70,50 @@ config NRF_CC310_BL
 	bool
 	default n
 
+if BOOT_USE_PSA_CRYPTO
+config BOOT_ED25519_PSA_DEPENDENCIES
+	bool
+	default n
+	select PSA_WANT_ALG_SHA_256
```

Comment:
PSA_WANT_ALG_SHA_256 is stranger there - won't be there once we will introduce corrected signature right?

Reply:
Yes, that would be with https://github.com/nrfconnect/sdk-mcuboot/pull/325 and https://github.com/mcu-tools/mcuboot/pull/2029